packages/nitro-protocol/test/contracts/AssetHolder/transfer.test.ts

```diff
@@ -122,6 +122,7 @@ describe('transfer', () => {
           }
         });
         const {events: eventsFromTx, gasUsed} = await (await tx).wait();
+        // NOTE: _transferAsset is a NOOP in TESTAssetHolder, so gas costs will be much lower than for a real Asset Holder
```

Comment:
Is it possible to "spend" some expected amount of gas spent transferring ERC20 tokens? eg. transferring a vanilla token to an address currently holding no tokens.

Reply:
That's possible, but I think this test is not the place to do that. We have other tests that target e.g. the `ERC20AssetHolder`, that do actually cover payouts to token holders and newbies.